doc/connectivity/resting_state_networks.rst

```diff
@@ -53,7 +53,9 @@ object.
 Visualizing the results
 ========================
 
-We can visualize the components as in the previous examples.
+We can visualize the components as in the previous examples. The first plot
+shows a map generated from all the components. Then we plot the activation of
+each component separately.
```

Comment:
Can you include the image here, using the 'image' directive, as elsewhere in the docs.